plugins/node/opentelemetry-instrumentation-graphql/src/instrumentation.ts

```diff
@@ -58,11 +58,14 @@ const DEFAULT_CONFIG: GraphQLInstrumentationConfig = {
   mergeItems: false,
   depth: -1,
   allowValues: false,
+  ignoreFields: [],
 };
 
 const supportedVersions = ['>=14'];
 
 export class GraphQLInstrumentation extends InstrumentationBase {
+  private _ignoreMap = new Map();
```

Comment:
this is always overridden in  `_setIgnoreMap` so I think it should be safe to remove assigning this to new map here ?

Reply:
Yep! You are definitely right, fixed it in the latest change